server/lib/host-metrics.js

```diff
@@ -266,6 +267,59 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(
+  host,
+  { startDate, endDate, collectiveIds = null, timeUnit } = {},
+) {
+  if (!collectiveIds) {
+    const results = await sequelize.query(`SELECT id FROM "Collectives" WHERE "HostCollectiveId" = :HostCollectiveId`, {
```

Comment:
This will include deleted collectives and the ones not approved yet (`isActive=false`). Let's make it simple (you'll need to update the `getHostedCollectives` function):

```suggestion
    const hostedCollectives = await host.getHostedCollectives({ attributes: ['id'] });
```

Reply:
Actually `getHostedCollectives` is pretty heavy as well. Feel free to just update your query above, but ignore deleted and inactive

Reply:
I've corrected it. Thanks. 👍🏽

Reply:
Ops, I just saw your second message. Let me adjust that. 😄

Reply:
Okay I've corrected it to include the `deletedAt` and `isActive` conditions. I realized we also do this same fetching of collectives here; 

https://github.com/opencollective/opencollective-api/blob/d198a0f09788381c3305f50cd4e0e84f2499c4c8/server/lib/budget.js#L178-L189 

And so I've corrected that with the same query as well.